src/env-inl.h

```diff
@@ -40,6 +40,31 @@
 
 namespace node {
 
+namespace loader {
+inline PackageConfig::PackageConfig(PackageConfig&& pkg)
+    : exists(pkg.exists),
+      is_valid(pkg.is_valid),
+      has_main(pkg.has_main),
+      main(pkg.main),
+      type(pkg.type),
+      has_exports(pkg.has_exports),
+      exports((v8::Global<v8::Value>&&) pkg.exports) {
+}
+
+inline PackageConfig::PackageConfig(Exists exists, IsValid is_valid,
+                                    HasMain has_main, const std::string& main,
+                                    PackageType type, HasExports has_exports,
+                                    v8::Global<v8::Value>&& exports)
+    : exists(exists),
+      is_valid(is_valid),
+      has_main(has_main),
+      main(main),
+      type(type),
+      has_exports(has_exports),
+      exports((v8::Global<v8::Value>&&) exports) {
```

Comment:
ditto,

```suggestion
      exports(std::move(exports)) {
```

Reply:
I think I had that code initially but ran into a similar "compiler confusion" as with the default implementation, it seems to want to use the deleted reference constructor for some reason:

```
In file included from ../src/base_object-inl.h:28:0,
                 from ../src/async_wrap-inl.h:28,
                 from ../src/api/callback.cc:2:
../src/env-inl.h: In constructor ‘node::loader::PackageConfig::PackageConfig(node::loader::PackageConfig&&)’:
../src/env-inl.h:51:37: error: use of deleted function ‘v8::Global<T>::Global(const v8::Global<T>&) [with T = v8::Value]’
       exports(std::move(pkg.exports)) {
                                     ^
In file included from ../src/node.h:63:0,
                 from ../src/api/callback.cc:1:
../deps/v8/include/v8.h:811:3: note: declared here
   Global(const Global&) = delete;
   ^~~~~~
```

Reply:
Would it help if I made it explicit which cast it is effectively? E.g.:

```cpp
      exports(const_cast<v8::Global<v8::Value>&&>(exports)) {
```

That seems to compile.